### PR TITLE
Prevent capturing of Network Info unless Perf is installed

### PR DESCRIPTION
**Pull request by samedson**
#no-changelog

### CODE REVIEW

File: FirebaseSessions/Sources/SessionStartEvent.swift [hunk 1]
```diff
@@ -57,13 +57,16 @@ class SessionStartEvent: NSObject, GDTCOREventDataObject {
     proto.application_info.which_platform_info = FIRSESGetAppleApplicationInfoTag()
     proto.application_info.apple_app_info
       .bundle_short_version = makeProtoString(appInfo.appDisplayVersion)
-    proto.application_info.apple_app_info.network_connection_info
-      .network_type = convertNetworkType(networkType: appInfo.networkInfo.networkType)
-    proto.application_info.apple_app_info.network_connection_info
-      .mobile_subtype = convertMobileSubtype(mobileSubtype: appInfo.networkInfo.mobileSubtype)
     proto.application_info.apple_app_info.os_name = convertOSName(osName: appInfo.osName)
     proto.application_info.apple_app_info.mcc_mnc = makeProtoString(appInfo.mccMNC)
```
Comment (visumickey) [inline FirebaseSessions/Sources/SessionStartEvent.swift:61]: Should mccMNC be also added as a reserved field? I assume Crashlytics does not capture this information?

**Review by leotianlizhan — APPROVED** [review]
lgtm

**Review by visumickey — APPROVED** [review]

File: FirebaseSessions/Sources/SessionStartEvent.swift (hunk 1 above)
Comment (samedson) [reply FirebaseSessions/Sources/SessionStartEvent.swift:61]: Yeah that totally makes sense to do 👍

Comment (samedson) [reply FirebaseSessions/Sources/SessionStartEvent.swift]: Done

**Pull request merged**